### DISCUSSION TITLE
WeakEventManager in MAUI

### DISCUSSION THREAD

**Discussion by symbiogenesis** [TheCodeTraveler/AsyncAwaitBestPractices#144] [Q&A] [answered]
WeakEventManager already exists in MAUI itself. Maybe this repo should direct people to use that, when on MAUI?

Or is there some other reason to prefer this one?

https://github.com/dotnet/maui/blob/main/src/Core/src/WeakEventManager.cs

**Comment by TheCodeTraveler** [comment, marked as answer]
> Or is there some other reason to prefer this one?

Yes. AsyncAwaitBestPractices' includes `WeakEventManager<T>`.
